src/index.js

```diff
@@ -5,12 +5,15 @@ import App from './App';
 import { AppContextProvider } from './context';
 import './index.css';
 import * as serviceWorker from './serviceWorker';
+import GAListener from './utils/analyticsUtils';
 
 ReactDOM.render(
   <React.StrictMode>
     <Router>
       <AppContextProvider>
-        <App />
+        <GAListener trackingId={process.env.REACT_APP_GA_TRACKERID}>
```

Comment:
@abeljoseph make sure we're aware of this when it comes time to deploy

Reply:
@atatmaja yup, already merged a [PR](https://github.com/serosurveillance-can-dev/sero-can-webapp/pull/107)  for this